runtime/near-vm-runner/src/memory.rs

```diff
@@ -8,11 +8,14 @@ pub struct WasmerMemory(Memory);
 
 impl WasmerMemory {
     pub fn new(config: &Config) -> Result<Self, VMError> {
-        Ok(WasmerMemory(Memory::new(MemoryDescriptor {
-            minimum: Pages(config.initial_memory_pages),
-            maximum: Some(Pages(config.max_memory_pages)),
-            shared: false,
-        })?))
+        Ok(WasmerMemory(
+            Memory::new(MemoryDescriptor {
+                minimum: Pages(config.initial_memory_pages),
+                maximum: Some(Pages(config.max_memory_pages)),
+                shared: false,
+            })
+            .expect("TODO creating memory cannot fail"),
```

Comment:
Why is it a TODO?

Reply:
it can fail on out of memory, TODO is to make sure it only fails because of node's fault